public/api/conf/1.0/testdata/get-pensions-dashboard.md

```diff
@@ -0,0 +1,178 @@
+             <td><p>Taxpayer who is affected by proposed change to State Pension age with COPE</p></td>
+             <td><p><code>ZX000063A</code></p></td>
+             <td><p>200 (OK) Payload as regular example above</p></td>
+         </tr>
+         <tr>
+             <td><p>Taxpayer who is not affected by proposed change to State Pension age with COPE</p></td>
+             <td><p><code>ZX000059A</code></p></td>
+             <td><p>200 (OK) Payload as regular example above</p></td>
+         </tr>  
+        <tr>
+            <th colspan="3">State Pension age exclusion</th>
+        </tr>
+        <tr>
+            <td><p>Taxpayer who has amount dissonance and is affected by proposed change to State Pension age</p></td>
+            <td><p><code>ZC974335B</code></p></td>
+            <td><p>200 (OK) Payload as exclusion example above</p></td>
+        </tr>
+        <tr>
+            <td><p>Taxpayer who has amount dissonance and is not affected by proposed change to State Pension age</p></td>
+            <td><p><code>ZC974325B</code></p></td>
+            <td><p>200 (OK) Payload as exclusion example above</p></td>
+        </tr>                
+         <tr>
+             <td><p>Taxpayer who has Isle of Man liability and is affected by proposed change to State Pension age</p></td>
+             <td><p><code>MA000004A</code></p></td>
+             <td><p>200 (OK) Payload as exclusion example above</p></td>
+         </tr>
+         <tr>
+             <td><p>Taxpayer who has Isle of Man liability and is not affected by proposed change to State Pension age</p></td>
+             <td><p><code>LG005101A</code></p></td>
+             <td><p>200 (OK) Payload as exclusion example above</p></td>
+         </tr>                          
+        <tr>
+            <th colspan="3">Exclusions</th>
+        </tr>
+        <tr>
+            <td><p>Taxpayer who has amount dissonance</p></td>
+            <td><p><code>ZC974325B</code></p></td>
+            <td><p>200 (OK) Payload as exclusion example above</p></td>
+        </tr>
+        <tr>
+            <td><p>Taxpayer who is post State Pension age</p></td>
+            <td><p><code>LG005901A</code></p></td>
+            <td><p>200 (OK) Payload as exclusion example above</p></td>
+        </tr>
+        <tr>
+            <td><p>Taxpayer who has Isle of Man liability</p></td>
+            <td><p><code>LG005101A</code></p></td>
+            <td><p>200 (OK) Payload as exclusion example above</p></td>
+        </tr>
+        <tr>
+            <td><p>Taxpayer who has the manual correspondence indicator flag</p></td>
+            <td><p><code>LG007101A</code></p></td>
+            <td><p>403 (Forbidden) {"code": "EXCLUSION_MANUAL_CORRESPONDENCE"}</p></td>
+        </tr>
+        <tr>
+            <td><p>Taxpayer who has a date of death</p></td>
+            <td><p><code>LG005801A</code></p></td>
+            <td><p>403 (Forbidden) {"code": "EXCLUSION_DEAD"}</p></td>
+        </tr>
+    </trs>
```

Comment:
no matching tag for TRS

Reply:
I believe this should be the closing tag for </tbody>

Reply:
tbody

Reply:
closed with </tbody> in place of </trs>